enableDetective.py

```diff
@@ -286,6 +286,22 @@ def accept_invitations(role: str, accounts: typing.Set[str], graph: str, region:
     except Exception as e:
         logging.exception(f'error accepting invitation {e.args}')
 
+def enable_detective(d_client: botocore.client.BaseClient, region: str):
+    graphs = get_graphs(d_client)
+    
+    if not graphs:
+        confirm = input('Should Amazon Detective be enabled in {}? Enter [Y/N]: '.format(region))
+
+        if confirm == 'Y':
```

Comment:
If you were nice you'd take 'y' as well.

Reply:
Updated :+1: